### DISCUSSION TITLE
Overlap between nmrCV and CHMO

### DISCUSSION THREAD

**Discussion by vcnainala** [NFDI4Chem/nmrxiv#92] [Ontologies]
todo

**Comment by StroemPhi** [comment]
import CHMO classes for NMR spectroscopy under Process/assay class in NMR CV [nmrCV#161](https://github.com/nmrML/nmrML/issues/161)

**Comment by StroemPhi** [comment]
[CHMO:nmr pulse sequence](https://terminology.nfdi4chem.de/ts/ontologies/chmo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHMO_0001841) & [nmrCV:NMR pulse sequence](https://terminology.nfdi4chem.de/ts/ontologies/nmrcv/terms?iri=http%3A%2F%2FnmrML.org%2FnmrCV%23NMR%3A1400037) including their subclasses

Here we might have a problem with the fact that nmrCV is aligned to BFO but not other OBO ontologies. The nmrCV:NMR pulse sequence is a bfo:ICE in CHMO it is a bfo:process. So they seem to me to represent the same thing but are classified differently. 

Then there is also an overlap wrt do the NMR devices. In CHMO we have ['nuclear magnetic resonance spectrometer'](https://terminology.nfdi4chem.de/ts/ontologies/chmo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHMO_0001807) and in nmrCV there is[ 'NMR instrument'](https://terminology.nfdi4chem.de/ts/ontologies/chmo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHMO_0001807), which has children that are needed in our use cases whereas CHMO doesn't.

**Comment by StroemPhi** [comment]
At the moment it seems to me, that both have overlapping scopes. And the taxonomic structure of nmrCV is quite different to CHMO as it is only aligned only to BFO not IAO or OBI. E.g. there is [nmrCV:instrument ](https://terminology.nfdi4chem.de/ts/ontologies/nmrcv/terms?iri=http%3A%2F%2FnmrML.org%2FnmrCV%23NMR%3A1000463&viewMode=All&siblings=false) which to me seems to be the same as [obi:device](https://terminology.nfdi4chem.de/ts/ontologies/chmo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FOBI_0000968&viewMode=All&siblings=false). So working with both at the same time can be very hard for the user and might lead to problems in an application context. With regard to the needed expressivity (ontological commitment) & axiomatisation, we need to take into account that nmrCV is only a taxonomy, whereas CHMO includes object properties to define classes. 

What I'm most interested in, is the about the question where to put new terms, when we need them and which of the two do I need to use in what contexts. What is the single point of truth for which apllication context?

I hope, this is helpful enough for further discussions.
